dts/bindings/rtc/nxp,kinetis-rtc.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Need to add license header

Reply:
Now fixed in https://github.com/zephyrproject-rtos/zephyr/pull/7182/commits/0fc81773e63f439238ab803e04eb55aa90dec4ab